configs/demo_config.yml

```diff
@@ -0,0 +1,45 @@
+extensions:
+  client:
+    endpoints:
+      - wss://cyber-testnet.alt.technology/ws
+  event_bus:
+  eth_api:
+    stale_timeout_seconds: 180 # rotate endpoint if no new blocks for 3 minutes
+  telemetry:
+    provider: none
+  cache:
+    default_ttl_seconds: 60
+    default_size: 500
+  server:
+    port: 8545
+    listen_address: '0.0.0.0'
+    max_connections: 2000
+    max_batch_size: 10
+    cors: all
+  whitelist:
+    eth_call_whitelist:
+      # allow 0x01 to call 0x02
+      - from: 0000000000000000000000000000000000000001
+        to: 0000000000000000000000000000000000000002
+      # allow anyone to call 0x01
+      - to: 0000000000000000000000000000000000000000
+      # allow 0x02 to create contract
+      - from: 0000000000000000000000000000000000000002
+        to: create
+    raw_tx_whitelist:
+      # allow 0x01 to call or create in raw tx.
+      - from: 0000000000000000000000000000000000000001
+    tx_whitelist:
```

Comment:
we could merge the whitelist of `eth_sendRawTransaction` and `eth_sendTransaction` RPCs, they are essentially the same.

Reply:
done